components/rich_text_editor/extensions/link/link.js

```diff
@@ -0,0 +1,36 @@
+import {
```

Comment:
This is the actual extension code, which is mostly showing that all the functionality comes from the ProseMirror plugin